lib/commands/addValue.js

```diff
@@ -56,7 +56,7 @@ let addValue = function (selector, value) {
         let elementIdValueCommands = []
 
         for (let elem of res.value) {
-            elementIdValueCommands.push(self.elementIdValue(elem.ELEMENT, value))
+            elementIdValueCommands.push(self.elementIdValue(elem[Object.keys(elem)[0]], value))
```

Comment:
Can we unify this operation? I've seen it couple of times in this PR and when the behavior changes we would need to go to each of these files. I think we can check the element value once before we make the request in the requestHandler.